zerver/lib/test_helpers.py

```diff
@@ -518,6 +502,8 @@ def find_pattern(pattern: Any, prefixes: List[str]) -> None:
             "scim/v2/Groups(?:/(?P<uuid>[^/]+))?",
             "scim/v2/Groups/.search",
             *(webhook.url for webhook in WEBHOOK_INTEGRATIONS if not include_webhooks),
+            # We exempt help center and api docs that have been renamed / redirected
+            *(redirect.old_url for redirect in DOCUMENTATION_REDIRECTS),
```

Comment:
I think there's also some number that we have actual tests for in `test_docs` or `test_urls`; can't remember which. 

I think it's possible we could just replace this with changing that code to just test them all -- testing a redirect should be very cheap.

Reply:
I could find no tests for any of these `redirect.old_urls` in the codebase. Should I write one that loops through the list?

Reply:
I found a test in `test_docs.py` for policy page redirects, so I used that as a template and wrote one that loops through the `DOCUMENTATION_REDIRECTS` for a success response and a common header in both the api and help center documentation ("Zulip homepage").

Reply:
Sounds good!